jsonschema-core.xml

```diff
@@ -2903,6 +2906,32 @@ https://example.com/schemas/common#/$defs/allOf/1
                     </t>
                 </section>
 
+                <section title="Dropped Annotations">
+                    <t>
+                        Any annotations produced and subsequently dropped by the evaluation
+                        due to an unsuccessful validation result of the containing subschema.
+                        This property MAY be included if the validation result of the containing
+                        subschema was unsuccessful.  It MUST NOT be included if the local
+                        validation result of the containing subschema was successful.
+                    </t>
+                    <t>
+                        Implementations that wish to include these annotations are encouraged
+                        to provide users with a configuration option to exclude them.
```

Comment:
Do we want this to be opt-in or opt-out?  It's potentially expensive, and because of `oneOf`, etc. it can happen on successful runs as well.

We don't need to resolve this to merge this PR, I'm just raising a philosophical point about what our default behavior ought to be, for both this and https://github.com/orgs/json-schema-org/discussions/236

Reply:
Yeah, I would expect this to be opt-in.  I'll add default behavior language.

Reply:
Updated.